Gopkg.toml

```diff
@@ -20,27 +19,37 @@
 #  name = "github.com/x/y"
 #  version = "2.4.0"
 
-
+ignored = ["k8s.io/client-go/pkg/api/v1"]
 [[constraint]]
-  name = "github.com/golang/glog"
+ name = "github.com/golang/glog"
 
 [[constraint]]
-  name = "github.com/kubernetes-incubator/external-storage"
-  version = "2.0.0"
+ name = "github.com/kubernetes-incubator/external-storage"
+ version = "1.8.1"
 
 [[override]]
-  name = "gopkg.in/gcfg.v1"
+ name = "gopkg.in/gcfg.v1"
 
 [[constraint]]
-  name = "k8s.io/apimachinery"
+ name = "k8s.io/apimachinery"
+ version = "kubernetes-1.9.1"
 
 [[constraint]]
-  name = "k8s.io/client-go"
+ name = "k8s.io/client-go"
+ version = "kubernetes-1.9.1"
 
 [[constraint]]
-  name = "k8s.io/kubernetes"
+ name = "k8s.io/kubernetes"
+ version = "1.9.1"
 
 [[constraint]]
-  version = "v1.0.0"
-  name = "github.com/oracle/oci-go-sdk"
-  
\ No newline at end of file
+ version = "v1.0.0"
+ name = "github.com/oracle/oci-go-sdk"
+
+[[override]]
+ name = "github.com/docker/distribution"
```

Comment:
Can we lock this down to latest version v2.6.2

Reply:
Rather than using branch master which may change, could we lock this down to a specific commit?